layers/meta-balena-jetson/recipes-core/images/resin-image.bbappend

```diff
@@ -20,3 +20,30 @@ device_specific_configuration_jetson-nano() {
       START=$(expr ${START} \+ ${NVIDIA_PART_OFFSET})
     done
 }
+
+
+# Binaries are signed and packed into
+# a partition and the flaser script
+# gets them from there. Can't store them
+# raw due to partition alignments which
+# trigger checksum mismatches during flash
+DEVICE_SPECIFIC_SPACE_jetson-xavier = "90112"
+
+do_image_resinos-img_jetson-xavier[depends] += " tegra194-flash-dry:do_deploy"
+device_specific_configuration_jetson-xavier() {
+    partitions=$(cat ${DEPLOY_DIR_IMAGE}/tegra-binaries/partition_specification194.txt)
```

Comment:
What is this variable used for? Seems not used?

Reply:
True, replaced by bootfiles, thanks for spotting it!